### PR TITLE
Support for Hitachi Storage

### PR DESCRIPTION
**Pull request by hille721**
Support for decoding NAA6 WWNs of Hitachi LUNs.

### CODE REVIEW

**Comment by jbrt** [comment]
Hello @hille721 

Thank you for this PR.
Can you please update the setup.py version with value 0.8.0 ?
Also, can you please add a unit test with some Hitachi WWN ?

Regards

**Review by jbrt — CHANGES_REQUESTED** [review]
Hello @hille721 

Here is the list of the waited changes 

- [ ] Renaming Hitachi class
- [ ] Updating setup.cfg version
- [ ] Provide a unit test for Hitachi WWN (I don't have the ability to test something on this kind of storage array, because I don't have one)

Regards

File: devicewwn/factory.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 from devicewwn.manufacturers.emc.vplex import EmcVplexWWN
 from devicewwn.manufacturers.ibm.npiv import IbmNpivWWN
 from devicewwn.manufacturers.netapp.netapp import NetappFasWWN
+from devicewwn.manufacturers.hitachi.hitachi import Hitachi
```
[the remark this answers is not in the document]
Comment (hille721) [reply devicewwn/factory.py]: oh, that was a typo
fixed now

**Comment by hille721** [comment]
@jbrt : everything done

**Comment by jbrt** [comment]
Hi again @hille721 

The version 0.8.0 has been upload on Pypi.
Thank you for your contribution.

Regards

**Pull request merged**